src/components/Cube.tsx

```diff
@@ -1,22 +1,16 @@
-import { useState } from "react";
-import { coord } from "../hooks/useGameState";
-
-export default function Cube(props: { position: coord }) {
-    // Hold state for hovered and clicked events
-    const [hovered, setHovered] = useState(false);
+import { Coord, Player } from "../hooks/useGameState";
 
+export default function Cube(props: { position: Coord, owner: Player }) {
     return (
         <mesh
             position={props.position}
             onPointerOver={(event) => {
                 event.stopPropagation();
-                setHovered(true);
             }}
             onClick={(e) => e.stopPropagation()}
-            onPointerOut={(_) => setHovered(false)}
         >
             <boxGeometry args={[1, 1, 1]} />
-            <meshStandardMaterial color={hovered ? "hotpink" : "orange"} />
+            <meshStandardMaterial color={props.owner === 'p1' ? "#000080" : "#008000"} />
```

Comment:
Are these colors likely to be reused elsewhere? If so you might consider storing them as const values.

Reply:
I'll take a look at this